k8s.io/README.md

```diff
@@ -3,6 +3,20 @@ Overview
 This contains the Nginx configuration for k8s.io and the associated subdomain
 redirectors.
 
+Redirections
```

Comment:
maybe add a comment in the ngnix configmap to keep this list up-to-date?

Reply:
Also link to the configmap so someone can find the source of truth?

Reply:
We should also document the vanity URLs we support (there are a lot of them, eg. pr.k8s.io)

Reply:
Ack @thockin i'll figure that out